implementations/rust/ockam/ockam_command/src/credential/store.rs

```diff
@@ -21,6 +21,9 @@ pub struct StoreCommand {
     #[arg(long = "issuer", value_name = "IDENTIFIER", value_parser = identity_identifier_parser)]
     pub issuer: Identifier,
 
+    #[arg(long = "scope", value_name = "CREDENTIAL_SCOPE")]
+    pub scope: String,
```

Comment:
Can you add some context to this argument explaining how/why is this used?

Reply:
Sure